pkg/gce-pd-csi-driver/controller.go

```diff
@@ -399,6 +403,10 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 		}
 		disk, err = createRegionalDisk(ctx, gceCS.CloudProvider, name, zones, params, capacityRange, capBytes, snapshotID, volumeContentSourceVolumeID, multiWriter)
 		if err != nil {
+			// Emit metric for error
+			if strings.Contains(params.DiskType, "pd") {
```

Comment:
Do we still need this diskType check? Because we are doing it in RecordOperationErrorMetrics

Reply:
Missed this disk type check, it is now removed